dashboard/app/controllers/levels_controller.rb

```diff
@@ -79,6 +79,7 @@ def update_blocks
   # PATCH/PUT /levels/1
   # PATCH/PUT /levels/1.json
   def update
+    level_params[:name] = level_params[:name].strip
```

Comment:
should we also do it on create? might be easier to do it in the Level model as a before_validation callback, eg:
http://stackoverflow.com/questions/3268053/rails-before-validation-strip-whitespace-best-practices